plugins/filter/decrypt.py

```diff
@@ -6,6 +6,97 @@
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
+DOCUMENTATION = '''
```

Comment:
I didn't realize that the sidecar docs changes also allow inline docs for filters (and tests?).

But I do wonder why you would choose to put them inline rather than in a separate file?

Reply:
Yes, that also works. Brian added two sidecar docs to community.general, one in this form and one with an extra YAML file, and both forms exist in ansible/ansible as well (IIRC).

> But I do wonder why you would choose to put them inline rather than in a separate file?

Because all other plugins/modules also do that (except PowerShell modules, and modules that are just action plugins with no module code :) ). Right now (not sure whether that won't change in the future) I prefer to keep the docs in the plugin's file if possible (which it is here).